docs/database-setup.md

```diff
@@ -1,51 +1,80 @@
 # Setting up a database
 
-PostgreSQL can be a bit of a pain to configure if you don't know where to begin.
-Basically, your goal is to create a database, create a user account that has
-access to that database, and allow access to that user account over a network
-connection (which means it must have a password).
+If you're reading this, you're taking on a role of DBA at this project.
+Congrats, now buckle up and put on your shit kickers.
 
-The following instructions have been tested to work on Ubuntu 16.04, but will
-probably work on other Debian-based distros too.
-
-## Installation
+## Local - Dev Only
+Okay, maybe you got scared of that DBA thing and you're just a dev.
+Do this:
 
     $ sudo apt-get install postgresql
-
-## Creating a user account
-
     $ sudo -u postgres createuser -P <username>
```

Comment:
This is (by tradition) the admin user as opposed to an application user.  Can we specify that here for clarity?  Even just a note about that.  Then this user will create other users.  I think in the later script that would make this "youwish_root"